web/public/locales/fr.json

```diff
@@ -1854,7 +1854,11 @@
     "title": "Paramètres"
   },
   "time-controller": {
-    "title": "Afficher les données passées"
+    "title": "Afficher les données passées",
+    "daily": "quotidien",
```

Comment:
```suggestion
    "daily": "par jour",
```

Reply:
Doesn't this mean "per hour" instead of "hourly"? I think we can do either way as @q-- mentioned, but I'd prefer it to be somewhat consistent across languages :)